src-tauri/src/schema.rs

```diff
@@ -39,9 +39,18 @@ pub struct FullItem {
     pub img_path: String,
     pub categories: Vec<Category>,
     pub rating: Rating,
+    pub traits: Vec<Trait>,
     pub date: NaiveDateTime,
 }
 
+#[derive(Eq, PartialEq, Default, Serialize, Deserialize, Clone, Debug)]
+pub struct Trait {
+    pub trait_id: usize,
+    pub name: usize,
+    pub range_low: String,
+    pub range_high: String,
```

Comment:
```suggestion
    pub name: String,
```

Reply:
Is there a reason to take out the range here?

Reply:
it twas a mistake of writing